cmd/podman/containers/cp.go

```diff
@@ -50,12 +50,17 @@ var (
 
 var (
 	cpOpts entities.ContainerCpOptions
+	chown  bool
 )
 
 func cpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 	flags.BoolVar(&cpOpts.Extract, "extract", false, "Deprecated...")
 	flags.BoolVar(&cpOpts.Pause, "pause", true, "Deprecated")
+	flags.BoolVarP(&chown, "archive", "a", false, `Archive mode (copy all uid/gid information).
+When set to true, files copied to a container will have changed ownership to the user of the container.
```

Comment:
How about:

"Maintain uid/gid from archive sources instead of changing them to the primary uid/gid of the container"

@TomSweeneyRedHat WDYT?